test/kcl/io_handler_tests.js

```diff
@@ -40,9 +39,13 @@ function captureStream(stream) {
 }
 
 describe('io_handler_tests', function() {
+  this.timeout(5000);
   var stdoutHook = null;
   var stderrHook = null;
-  var ioHandler = new IOHandler(process.stdin, process.stdout, process.stderr);
+  // Github workflows doesn't write to process.stdin for unknown reasons, so using a new Stream
+  const readableStream = new Stream.Readable();
+  readableStream._read = () => {}; // _read is required but can noop it
```

Comment:
What is it that is happening here?

Reply:
Writing to stdin doesn't work on workflow but works locally. So I create a separate stream object similar to stdin stream. I think maybe github workflows is doing something with stdin

Reply:
I assume this also works with local